supabase/migrations/20240425205531_create_auth_challege_table.sql

```diff
@@ -0,0 +1,36 @@
+create table "public"."challenges" (
+    "id" serial primary key,
+    "challenge" bytea not null,
+    "created_at" timestamp with time zone not null default current_timestamp,
+    "expires_at" timestamp with time zone not null default current_timestamp + interval '15 minute',
+    unique (challenge)
+);
+
+
+alter table "public"."challenges" enable row level security;
+grant all on table "public"."challenges" to service_role;
```

Comment:
shouldn't be required.
```suggestion
grant all on table "public"."challenges" to service_role;
```